launch_testing/launch_testing/junitxml.py

```diff
@@ -83,9 +83,10 @@ def unittestCaseToXml(test_result, test_case):
     class needs to be a launch_testing.TestResult class
     """
     case_xml = ET.Element('testcase')
-    full_classname = type(test_case).__module__ + '.' + type(test_case).__name__
+    full_methodname, _, qualifiers = test_case.id().partition(' ')
```

Comment:
can you give an example of how a full method name with qualifiers look?

Reply:
Qualifiers apply to the test case, not the method. When using the [`subTest` feature](https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest), `unittest` appends message and parameters that I need to account for.

Reply:
thanks for clarifying, sounds good